sound/soc/sof/core.c

```diff
@@ -390,6 +391,9 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 		goto dbg_err;
 	}
 
+	/* optionally register virtio miscdev */
+	dsp_sof_virtio_miscdev_register(sdev);
+
```

Comment:
conditionaly compiled code ?

Reply:
`dsp_sof_virtio_miscdev_register()` is replaced by a dummy if VHOST SOF isn't selected, yes, that shouldn't be a problem?

Reply:
@lgirdwood a vBE, but that probably has long stopped being a question, especially with The Doc